### PR TITLE
GAWB-1031: Show submission status icons in submissions list

### PR DESCRIPTION
**Pull request by MatthewBemis**
rawls piece: https://github.com/broadinstitute/rawls/pull/541

~~I want to try it out again but can't run it all together since the local SSL certs are expired atm~~
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: JIRA ticket checks:
  - Acceptance criteria exists and is met
  - Note any changes to implementation from the description
  - Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  - Authentication
  - Authorization
  - Encryption
  - Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell the tech lead (TL) that the PR exists if they wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- Review cycle:
  - LR reviews
  - Rest of team may comment on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH
  - Submitter updates documentation as needed
  - Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs [hunk 1]
```diff
@@ -58,6 +58,13 @@
     :else (do (utils/log "Unknown workflow status: " status)
             failure-icon)))
 
+(defn icon-for-sub-status [wf-statuses]
+  (cond
+    (contains? wf-statuses "Failed") failure-icon
+    (contains? wf-statuses "Succeeded") success-icon
+    :else (do (utils/log "Unknown submission status")
+            nil)))
```
Comment (MatthewBemis) [inline src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs]: is nil safe to return here? i don't want to display an icon if the submission isn't in either of the first two states

**Comment by davidangb** [comment]
Can't really review this without knowing what the payload from rawls is going to be. Is workflowStatuses a string, array, object, etc?

Related - update the mock-data inside endpoints/list-submissions to include workflowStatuses?

**Comment by MatthewBemis** [comment]
@davidangb here's the rawls piece which should be pretty stable: https://github.com/broadinstitute/rawls/pull/541

workflowStatuses is a Map[String, Int] that looks something like:

```
{
  Failed: 10,
  Succeeded: 5,
  Running: 5
}
```

**Comment by davidangb** [comment]
Looks good from reading the code. I'd like to try it, but am also stymied by SSL certs. Will need a rebase, I assume because of #521.

**Comment by davidangb** [comment]
tried it locally (and committed a minor fix). 👍 assuming you rebase.

**Comment by skwalker** [comment]
LGTM 👍

**Pull request merged**